Tests/SentryTests/Protocol/SentryLogAttributeTests.swift

```diff
@@ -0,0 +1,201 @@
+@testable import Sentry
+import XCTest
+
+final class SentryLogAttributeTests: XCTestCase {
+    
+    // MARK: - Encoding Tests
+    
+    func testEncodeStringAttribute() throws {
+        let attribute = SentryLog.Attribute.string("test value")
+        
+        let data = try JSONEncoder().encode(attribute)
+        let json = try JSONSerialization.jsonObject(with: data) as? [String: Any]
+        
+        XCTAssertNotNil(json)
+        XCTAssertEqual(json?["type"] as? String, "string")
+        XCTAssertEqual(json?["value"] as? String, "test value")
+    }
```

Comment:
`l`: You can use `XCTUnwrap` including a cast to `[String:String]` to avoid having to use `XCTAssertNotNil` and casting again in `XCTAssertEqual` to simplify the tests. This applies to all encoding tests.
```suggestion
    func testEncodeStringAttribute() throws {
        let attribute = SentryLog.Attribute.string("test value")
        
        let data = try JSONEncoder().encode(attribute)
        let json = try XCTUnwrap(JSONSerialization.jsonObject(with: data) as? [String: String])

        XCTAssertEqual(json["type"], "string")
        XCTAssertEqual(json["value"], "test value")
    }
```